README.md

```diff
@@ -230,6 +231,20 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
 
   We also have a dev slack channel, please follow this [link](https://join.slack.com/t/ai-habitat/shared_invite/enQtNjY1MzM1NDE4MTk2LTZhMzdmYWMwODZlNjg5MjZiZjExOTBjOTg5MmRiZTVhOWQyNzk0OTMyN2E1ZTEzZTNjMWM0MjBkN2VhMjQxMDI) to get added to the channel.
 
+## WebGL
+
+1. Download the [test scenes](http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip) and extract locally to habitat-sim creating habitat-sim/data.
```

Comment:
Maybe directly provide `wget` based command?

Reply:
wget is not available on Mac by default.
I'm also not sure that is easier than downloading a link off the web-site.